src/current/_data/releases.yml

```diff
@@ -4867,3 +4867,29 @@
     docker_arm_limited_access: false
   source: true
   previous_release: v23.1.9
+
+- release_name: v23.2.0-alpha1
```

Comment:
@ianjevans I think this is a typo during the RN script invocation

I think this should read v23.2.0-alpha.1